ljconf.go

```diff
@@ -162,6 +166,36 @@ func Load(fn string) (conf *Conf, err error) {
 	return conf, nil
 }
 
+// Watch periodically checks the configure file in curConf with the specified interval.
+// If the configuration file changes, it's reloaded and sent to the specified channel as a *Conf.
+// TODO use inotify mechanism instead of poll.
+func Watch(curConf *Conf, interval time.Duration, ch chan *Conf) error {
+	ticker := time.NewTicker(interval)
+	defer ticker.Stop()
+
+	configFileName := string(curConf.path)
+	for _ = range ticker.C {
+		stat, err := os.Stat(configFileName)
+		if err != nil {
+			// e,g. the config file was deleted
+			return err
+		}
+
+		if stat.ModTime() != curConf.lastStat.ModTime() {
+			curConf.lastStat = stat
```

Comment:
I think `curConf.lastStat` is not supposed to change since contents inside it don't change. Maybe we should define `lastStat` as a local variable.